src/Fl_Widget.cxx

```diff
@@ -133,7 +133,84 @@ Fl_Widget::Fl_Widget(int X, int Y, int W, int H, const char* L) {
   if (Fl_Group::current()) Fl_Group::current()->add(this);
 }
 
+/** Updates the size and position of widgets using hints and requests.
+
+ parametric_resize() handles additional information in the width and height
+ parameters of the resize request. Size request are created using predifined
```

Comment:
Size request**s** are created using pred**e**fined ...
Note: TWO typos in this line.